website/content/docs/upgrading/upgrade-to-1.16.x.mdx

```diff
@@ -61,6 +61,26 @@ If you register an external plugin called `myplugin` with `SOURCE=child`, the
 plugin process starts with `SOURCE=parent` and Vault reports a conflict for
 `myplugin`.
 
+### LDAP auth entity alias names no longer include upndomain
+
+The `userattr` field on the LDAP auth config is now used as the entity alias.
+Prior to 1.16, the LDAP auth method would detect if `upndomain` was configured
+on the mount and then use `<cn>@<upndomain>` as the entity alias value.
+
+The consequence of not configuring this correctly means users may not have the
+correct policies attached to their tokens when logging in.
```

Comment:
```suggestion
The consequence of not configuring this correctly means entity alias names will 
change between 1.15 and 1.16, which may result in additional aliases or missing policies 
on tokens.
```

Reply:
Thanks! I had automerge enabled but I will amend this in a follow up PR.